src/lib/homestore_backend/replication_state_machine.cpp

```diff
@@ -71,6 +71,8 @@ void ReplicationStateMachine::on_rollback(int64_t lsn, sisl::blob const& header,
     }
 }
 
+void ReplicationStateMachine::on_restart() { home_object_->on_replica_restart(); }
```

Comment:
should we add `std::call_once` ?  since the on_restart will be called for each repl_dev but on_replica_restart only need to run one time.

Reply:
Yes, you are right. We need call_once